services-js/payment-webhooks/server/stripe-events.ts

```diff
@@ -1,20 +1,27 @@
 /* eslint no-console: 0 */
 
-import { NodeStripe, Event, Charge } from 'stripe';
+import Stripe, { webhooks, charges } from 'stripe';
 import INovah from './services/INovah';
 
 import Rollbar from 'rollbar';
 
 type Dependencies = {
   inovah: INovah;
   rollbar: Rollbar;
-  stripe: NodeStripe;
+  stripe: Stripe;
 };
 
 async function processChargeSucceeded(
   { rollbar, stripe, inovah }: Dependencies,
-  charge: Charge
+  charge: charges.ICharge
 ): Promise<void> {
+  // Birth cert charges aren't captured at first, we wait until fulfillment to
```

Comment:
Would it make sense to expand on this comment to make clear that fulfillment is a human-triggered event?

Reply:
Updated.

Reply:
awesome, thanks!